template.yml

```diff
@@ -185,7 +185,10 @@ Resources:
           # The Logical ID of the BuildAndRun Lambda Alias, which is generated by SAM because AutoPublishAlias is enabled
           # has a predictable format: <function‑LogicalId>Alias<alias‑name>
           # https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-generated-resources-function.html#sam-specification-generated-resources-function-autopublishalias
-          BUILD_AND_RUN_PROJECT_LAMBDA_ARN: !Ref BuildAndRunJavaProjectFunctionAliaslive
+          BUILD_AND_RUN_THEATER_PROJECT_LAMBDA_ARN: !Ref BuildAndRunJavaTheaterProjectFunctionAliaslive
+          BUILD_AND_RUN_NEIGHBORHOOD_PROJECT_LAMBDA_ARN: !Ref BuildAndRunJavaNeighborhoodProjectFunctionAliaslive
+          BUILD_AND_RUN_PLAYGROUND_PROJECT_LAMBDA_ARN: !Ref BuildAndRunJavaPlaygroundProjectFunctionAliaslive
+          BUILD_AND_RUN_CONSOLE_PROJECT_LAMBDA_ARN: !Ref BuildAndRunJavaConsoleProjectFunctionAliaslive
```

Comment:
Note: I think we'll want to adjust the names here a bit to make them easier to use in mappings elsewhere. i.e. something like this https://github.com/code-dot-org/code-dot-org/blob/004deda4d0d5408904a76eaec5c1345f190abbde/aws/cloudformation/components/alarms.yml.erb#L24